config/core/configmaps/features.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   labels:
     serving.knative.dev/release: devel
   annotations:
-    knative.dev/example-checksum: 43bdc81c
+    knative.dev/example-checksum: 52231144
```

Comment:
I think this should be quoted - the yaml interpretors think it's a number which breaks the checksum.

Reply:
Thank you! I was so confused about why it wasn't interpreting that!

...at first glance I thought something was wrong with the entirely numeric checksum. Just lucky I guess.